libs/@guardian/source-foundations/src/typography/data.ts

```diff
@@ -90,8 +90,8 @@ export const fonts = {
 
 export const lineHeights = {
 	tight: 1.15,
-	regular: 1.35,
-	loose: 1.5,
+	regular: 1.3,
+	loose: 1.4,
```

Comment:
Worth noting that none of these meet the [WCAG 2.1 AA Success Criterion for Text Spacing](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing)